package.json

```diff
@@ -28,6 +28,7 @@
     "realclean": "rm -rf dist build node_modules web_modules",
     "start": "snowpack dev --polyfill-node",
     "build": "snowpack build --polyfill-node",
+    "fix": "snowpack --reload --polyfill-node",
```

Comment:
I don't quite understand what this is about. Care to elaborate?

Not critical. It looks reasonably harmless.